Local Pods/GitHubAPI/GitHubAPI/V3DiscissionsRequest.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  V3DiscissionsRequest.swift
+//  GitHubAPI
+//
+//  Created by Bas Broek on 28/07/2018.
+//  Copyright © 2018 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+public struct V3DiscussionsRequest: V3Request {
+    public typealias ResponseType = V3DataResponse<[V3Discussion]>
+    public var pathComponents: [String] {
+        return ["teams", teamID, "discussion"]
+    }
+
+    public var parameters: [String : Any]? {
+        return ["per_page": 50, "page": page, "direction": direction.rawValue]
+    }
+
+    public var headers: [String : String]? {
+        return ["Accept": "application/vnd.github.echo-preview+json"]
+    }
+
+    public enum Direction: String {
```

Comment:
And we might want to abstract this to somewhere else so it can be reused — I think it is supported by some other requests as well.